### PR TITLE
Demo changes 

### PR DESCRIPTION
**Pull request by bhanvimenghani**
contains changes made to box plot axis values for the demo.

### CODE REVIEW

File: src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationTables.tsx [hunk 1]
```diff
@@ -18,10 +18,12 @@ import { WorkloadDetails } from './RecommendationComponents/WorkloadDetails';
...
   // @ts-ignore
-  const list_recommendations_url: string = getRecommendationsURLWithParams(
-    sessionStorage.getItem('Experiment Name') || '',
-    'false'
-  );
+  // const list_recommendations_url: string = getRecommendationsURLWithParams(
+  //   sessionStorage.getItem('Experiment Name') || '',
+  //   'false'
+  // );
+  const list_recommendations_url: string = 'https://mocki.io/v1/9dd25d7b-2d79-4a3b-b586-0a767e5dbf41'
```
Comment (dinogun) [inline src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationTables.tsx:25]: Is this intentional?

Comment (bhanvimenghani) [reply src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationTables.tsx:25]: thanks for pointing out, i will fix this

**Comment by bhanvimenghani** [comment]
latest quay image` quay.io/bmenghan/demo_changes`

**Review by dinogun — APPROVED** [review]
LGTM!

**Pull request merged**